application.py

```diff
@@ -3,12 +3,16 @@
 
 from flask import Flask, session,render_template, request,flash, redirect, url_for
 from flask_session import Session
+from flask_humanize import Humanize
```

Comment:
what is this humanize for?

Reply:
humanize is for day display of the review.

Reply:
can you elaborate ?

Reply:
Humanize is a flask module which changes the date and time format in which we wish to display.
so I have used this module in displaying reviews mentioning when the review is posted by user like Today, Tomorrow, Yesterday.